docs/documentation/installation-guide.md

```diff
@@ -1,4 +1,143 @@
+--max_iterations <int [1,inf[>
+                              Max iteration number, also the max possible number of attributs in a rule, should be 25 if working with images (default: 10)
+--min_covering <int [1,inf[>  Minimum covering number (default: 2)
+--covering_strategy <bool>    Whether to use this strategy : if no rule is found with min_covering, find best rule with best covering using dichotomic search. Decreases min_fidelity if needed (default: True)
+--max_failed_attempts <int [0,inf[>
+                              Maximum number of failed attempts to find Fidex rule when covering is 1 and covering strategy is used (default: 30)
+--min_fidelity <float [0,1]>  Minimal rule fidelity accepted when generating a rule (default: 1.0)
+--lowest_min_fidelity <float [0,1]>
+                              Minimal min_fidelity to which we agree to go down during covering_strategy (default: 0.75)
+--dropout_dim <float [0,1]>   Dimension dropout parameter (default: 0.0)
+--dropout_hyp <float [0,1]>   Hyperplan dropout parameter (default: 0.0)
+--nb_quant_levels <int [3,inf[>
+                              Number of stairs in staircase activation function (default: 50)
+--decision_threshold <float [0,1]>
+                              Decision threshold for predictions, you need to specify the index of positive class if you want to use it
+--positive_class_index <int [0,nb_classes-1]>
+                              Index of positive class for the usage of decision threshold, index starts at 0
+--normalization_file <str>    File containing the mean and std of some attributes. Used to denormalize the rules if specified
+--mus <list<float ]inf,inf[>> Mean or median of each attribute index to denormalize in the rules
+--sigmas <list<float ]inf,inf[>>
+                              Standard deviation of each attribute index to denormalize in the rules
+--normalization_indices <list<int [0,nb_attributes-1]>>
+                              Attribute indices to denormalize in the rules, only used when no normalization_file is given, index starts at 0 (default: [0,...,nb_attributes-1])
+--nb_threads <int [1,nb_cores]>
+                              Number of threads used for computing the algorithm, 1=sequential execution (default: 1)
+--seed <int [0,inf[>          Seed, 0=random (default: 0)
+
+----------------------------
+
+Execution example :
+
+fidex.fidexGloRules("--train_data_file datanormTrain.txt --train_pred_file predTrain.out --train_class_file dataclass2Train.txt --weights_file weights.wts --nb_attributes 16 --nb_classes 2 --heuristic 1 --global_rules_outfile globalRules.rls --root_folder dimlp/datafiles")
+
+---------------------------------------------------------------------
+```
+
+You are now set to use all `Dimlpfidex` tools! You can continue by following the [getting-started tutorial](../getting-started) to learn more about how to use the different tools available.
+
+## Installation by compiling sources (alternative)
```

Comment:
I'd remove thsi section entirely. People that really want to compile have access to the code repository and README. For the rest of the people, it's background noise. Note: Nobody should compile manually anyway since this is such a mess anyway.